Makefile

```diff
@@ -201,6 +202,23 @@ download_or_build_containers:
 # Update databases
 # ==============================================================================
 
+# -----------------------------
+# Download Hostile human index
+# -----------------------------
+# HOSTILE_DIR := $(ASSETS_DIR)/hostile_db
+# update_hostile_db: $(HOSTILE_DIR)/human-t2t-hla.fa.gz
+
+# $(HOSTILE_DIR)/human-t2t-hla.fa.gz:
```

Comment:
I'm actually gonna manually download the genome and bowtie & minimapper index it myself.